ivy_tests/test_ivy/test_frontends/test_torch/test_pointwise_ops.py

```diff
@@ -2081,3 +2081,33 @@ def test_torch_sigmoid(
         on_device=on_device,
         input=x[0],
     )
+
+
+# lerp
+@handle_frontend_test(
+    fn_tree="torch.lerp",
+    dtype_and_input=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("float"),
+        num_arrays=3,
+    ),
+)
+def test_torch_lerp(
+    *,
+    dtype_and_input,
+    frontend,
+    test_flags,
+    fn_tree,
+    on_device,
+):
+    input_dtype, inputs = dtype_and_input
+    start, end, weight = inputs
+    helpers.test_frontend_function(
+        input_dtypes=input_dtype,
+        frontend=frontend,
+        test_flags=test_flags,
+        fn_tree=fn_tree,
+        on_device=on_device,
+        input_start=start,
+        input_end=end,
+        input_weight=weight,
```

Comment:
just have this be `input=start, end=end` and so on, the kwargs must have the same names as the function args